scripts/dnd5e/AttackPreper.js

```
@@ -249,6 +248,8 @@ export default class AttackPreper extends ItemPreper {
 	 */
 	damageFormula(index=0) {	// Extract and re-format the damage formula
 		const roll = new Roll(this.getAttackFormula(index), this.item.getRollData());
+		var dice = game[game.system.id].dice;
+		var simplifyRollFormula = dice?.simplifyRollFormula;
```

Do not use `var`. Use only `let` or `const`. In this case, the appropriate choice is `const`.

Since this function isn't being used multiple times here though, I would probably just change the last three lines to:
```js
return game[game.system.id].dice?.simplifyRollFormula(roll.formula);
```
Though... is there *actually* a chance that `dice` doesn't exist? If so, we might need to add something to handle that case and take an alternate path. If it can't really fail to exist, then we should remove the `?.` so that we don't eat an error if something really weird happens.